### PR TITLE
add node example to open graph image docs

### PR DESCRIPTION
**Pull request by souporserious**
### What?

Adds a Node.js example for using assets with `ImageResponse` to create og images.

### Why?

The only examples available use `new URL` and `fetch` which does not work when the runtime is using Node.js.

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: c2a04704dfac3aa3069b27d65e7d2b810060a275

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: docs/02-app/02-api-reference/02-file-conventions/01-metadata/opengraph-image.mdx [hunk 1]
```diff
@@ -421,6 +421,34 @@ export default async function Image({ params }) {
 }
 ```
 
+#### Using Node runtime with local assets
+
+This example uses the Node runtime to fetch a local image on the file system and pass it as an `ArrayBuffer` to an `img` element `src`:
```
Comment (delbaoliveira) [inline docs/02-app/02-api-reference/02-file-conventions/01-metadata/opengraph-image.mdx:426]:
```suggestion
This example uses the Node.js runtime to fetch a local image on the file system and pass it as an `ArrayBuffer` to an `src` attribute of an `<img>` element. 
```

File: docs/02-app/02-api-reference/02-file-conventions/01-metadata/opengraph-image.mdx [hunk 2]
```diff
@@ -421,6 +421,34 @@ export default async function Image({ params }) {
 }
 ```
 
+#### Using Node runtime with local assets
```
Comment (delbaoliveira) [inline docs/02-app/02-api-reference/02-file-conventions/01-metadata/opengraph-image.mdx:424]:
```suggestion
#### Using Node.js runtime with local assets
```

**Review by delbaoliveira — CHANGES_REQUESTED** [review]
Thank you @souporserious, while we're here, could we also add the Edge runtime example: https://vercel.com/docs/functions/og-image-generation/og-image-examples#using-a-local-image

This is less about the runtime, but more on how to import local images (using the different runtimes).

**Comment by souporserious** [comment]
@delbaoliveira yeah, no problem! Updated the examples to showcase Edge runtime as well 😊

**Review by delbaoliveira — APPROVED** [review]
Thank you @souporserious 🙏🏼

**Pull request merged**